port_ocean/core/integrations/mixins/sync_raw.py

```diff
@@ -184,7 +184,7 @@ async def _register_in_batches(
         send_raw_data_examples_amount = (
             SEND_RAW_DATA_EXAMPLES_AMOUNT if ocean.config.send_raw_data_examples else 0
         )
-        all_entities, register_errors = await self._register_resource_raw(
+        all_entities, register_errors,_ = await self._register_resource_raw(
```

Comment:
isn't this a lint error?

Reply:
lint passes :) 
Thought so too at first ^_^